gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/ClassUniquenessAnalyzer.java

```diff
@@ -76,6 +76,12 @@ public void analyzeConfiguration(Configuration configuration) {
                         continue;
                     }
 
+                    if (entry.getName().equals("module-info.class")) {
```

Comment:
maybe package-info too? 
https://github.com/nebula-plugins/gradle-lint-plugin/blob/master/src/main/kotlin/com/netflix/nebula/lint/rule/dependency/DuplicateDependencyService.kt#L11-L16

Reply:
So that one didn't make quite so much sense to me.  It's perfectly legitimate in Java9 to have root level module-info files, but I think identically named package-info ones is still bad (because these could be annotated differently and could therefore clash... )

I'd be happy to add that special-case once we find a real counter example!